.travis.yml

```
@@ -40,6 +40,9 @@ matrix:
       env: SYMFONY_REQUIRE=4.4.*
     - php: 7.4
       env: SYMFONY_REQUIRE=5.0.*
+      # PHP 8.0 (Travis does not yet support using the stable 8.0.0)
+    - php: nightly
+      env: SYMFONY_REQUIRE=5.2.*
```

```suggestion
      env: SYMFONY_REQUIRE=5.2.* COMPOSER_FLAGS="--ignore-platform-req=php"
```

Should we ignore the platform check temporarily to see if everything else works out?